src/SearchParameters/index.js

```diff
@@ -1217,9 +1216,11 @@ SearchParameters.prototype = {
    */
   getRefinedDisjunctiveFacets: function getRefinedDisjunctiveFacets() {
     // attributes used for numeric filter can also be disjunctive
-    var disjunctiveNumericRefinedFacets = intersection(
-      keys(this.numericRefinements),
-      this.disjunctiveFacets
+    var numericRefinements = Object.keys(this.numericRefinements);
+    var disjunctiveNumericRefinedFacets = this.disjunctiveFacets.filter(
```

Comment:
I think I would have factorised this intersection logic in a function an call it `intersection`, as it's much clearer to understand what's happening like that I believe.

Reply:
I disagree, I find both usages without the helper function much easier. You might be biased because that's the way it was written before, or because you're very familiar with lodash.

I'm also trying to avoid new helper functions here, because they will quickly become as complicated as the lodash ones, and then we won nothing by removing it :)

Reply:
I wouldn't say `nothing`. 
The code you would have created is much more concise than the [original] (https://github.com/lodash/lodash/blob/master/.internal/baseIntersection.js), leave alone the dependencies it's using 

It's fine to leave it like that